### PR TITLE
Adding counts to histogram

### LINKED ISSUE #5343: Histograms to show proportions or percentages
Currently our histograms show counts (frequencies).  How can we use our dialogues to show proportions or percentages.

There are questions on stack-overflow , for example [here](https://stackoverflow.com/questions/3695497/show-instead-of-counts-in-charts-of-categorical-variables).  

The solution appears to be to add a y to the aes.  I give an example of the additional code below:

It is just the bit in bold that needs to be added, i.e. **y = (..count..)/sum(..count..)**  The rest is easy from the dialogue.  If you multiply by 100 you get percentages and alternatively that can be done with scale_y_continuous(labels = percent).

last_graph <- ggplot2::ggplot(data=mydata, mapping=ggplot2::aes(x=wd1,**y = (..count..)/sum(..count..)**)) + ggplot2::geom_histogram(fill="blue", boundary=-5, bins=36) + theme_grey()+coord_polar(start=0)+ ggplot2::scale_y_continuous(limits=c(-2, 10))

But how can we do this sort of stuff from the dialogue, rather than the script?

### PR DESCRIPTION
**Pull request by Wycklife**
Fixes #5343 

@maxwellfundi This has now been addrressed!

### CODE REVIEW

**Comment by maxwellfundi** [comment]
Great @rdstern the histograms are now merged.

File: instat/dlgHistogram.vb [hunk 1]
```diff
@@ -280,4 +321,16 @@ Public Class dlgHistogram
...
+
+    End Sub
+
+    Private Sub ucrVariablesAsFactorforHist_SelectionChanged(sender As Object, e As EventArgs) Handles ucrVariablesAsFactorforHist.SelectionChanged
+
+    End Sub
+
+    Private Sub ucrPnlOptions_Control(ucrChangedControl As ucrCore) Handles ucrPnlOptions.ControlValueChanged
+
+    End Sub
```
Comment (dannyparsons) [inline instat/dlgHistogram.vb]: @Wycklife please remove these empty subs in a new pull request.

File: instat/frmMain.vb [hunk 2]
```diff
@@ -1773,11 +1773,11 @@ Public Class frmMain
     End Sub
 
 
-    Private Sub mnuProcurementCalculateCRI_Click(sender As Object, e As EventArgs) Handles mnuProcurementCalculateCRI.Click
+    Private Sub mnuProcurementCalculateCRI_Click(sender As Object, e As EventArgs) Handles mnuProcurementCTFVCalculateCRI.Click
         dlgCorruptionDefineCRI.ShowDialog()
     End Sub
 
-    Private Sub mnuProcurementDefineCorruption_Click_1(sender As Object, e As EventArgs) Handles mnuProcurementDefineCorruption.Click
+    Private Sub mnuProcurementDefineCorruption_Click_1(sender As Object, e As EventArgs) Handles mnuProcurementModelDefineCorruption.Click
```
Comment (dannyparsons) [inline instat/frmMain.vb]: @Wycklife Please avoid making changes to frmMain in a pull request for another dialog in future.

Comment (Wycklife) [reply instat/frmMain.vb]: @dannyparsons Noted

File: instat/dlgHistogram.vb (hunk 1 above)
Comment (Wycklife) [reply instat/dlgHistogram.vb]: @dannyparsons I've made the changes but where do they appear being that the issue was merged?

**Pull request merged**